app/src/main/kotlin/com/simplemobiletools/calendar/extensions/Context.kt

```diff
@@ -184,9 +184,17 @@ fun Context.getNotification(pendingIntent: PendingIntent, event: Event, content:
     val soundUri = Uri.parse(config.reminderSoundUri)
     grantReadUriPermission(config.reminderSoundUri)
 
+    val contentTitle = when (publicVersion) {
+        false -> event.title
+        true -> resources.getString(R.string.app_name)
+    }
```

Comment:
oh gosh, rewrite this to "val contentTitle = if (publicVersion) resources.getString(R.string.app_name) else event.title" please

Reply:
no problem